WooCommerce/Classes/Authentication/Jetpack Setup/Site Credential Login/SiteCredentialLoginView.swift

```diff
@@ -0,0 +1,199 @@
+                JetpackInstallHeaderView()
+
+                // title and description
+                VStack(alignment: .leading, spacing: Constants.contentVerticalSpacing) {
+                    Text(Localization.title)
+                        .largeTitleStyle()
+                    AttributedText(descriptionAttributedString)
+                }
+
+                // text fields
+                VStack(alignment: .leading, spacing: Constants.contentVerticalSpacing) {
+                    VStack(alignment: .leading, spacing: Constants.fieldVerticalSpacing) {
+                        TextField(Localization.enterUsername, text: $viewModel.username)
+                            .textFieldStyle(.plain)
+                            .focused($keyboardIsShown)
+                        Divider()
+                    }
+                    .frame(height: Constants.fieldHeight * scale)
+
+                    VStack(alignment: .leading, spacing: Constants.fieldVerticalSpacing) {
+                        if showsSecureInput {
+                            TextField(Localization.enterPassword, text: $viewModel.password)
+                                .textFieldStyle(.plain)
+                                .focused($keyboardIsShown)
+                        } else {
+                            SecureField(Localization.enterPassword, text: $viewModel.password)
+                                .focused($keyboardIsShown)
+                        }
+                        Divider()
+                    }
+                    .frame(height: Constants.fieldHeight * scale)
+                    .overlay(HStack {
+                        Spacer()
+                        // Button to show/hide the text field content.
+                        Button(action: {
+                            showsSecureInput.toggle()
+                        }) {
+                            Image(systemName: showsSecureInput ? "eye.slash" : "eye")
+                                .accentColor(Color(.textSubtle))
+                                .frame(width: Constants.eyeButtonDimension * scale,
+                                       height: Constants.eyeButtonDimension * scale)
+                                .padding(.horizontal, Constants.eyeButtonHorizontalPadding)
+                        }
+                    })
+                }
+
+                Label {
+                    Text(Localization.note)
+                } icon: {
+                    Image(systemName: "info.circle")
+                }
+                .foregroundColor(Color(uiColor: .secondaryLabel))
+
+                Spacer()
+            }
+        }
+        .safeAreaInset(edge: .bottom, content: {
+            Button {
+                // TODO-8075: add tracks
+                keyboardIsShown = false
+                viewModel.handleLogin()
+            } label: {
+                Text(connectionOnly ? Localization.connectJetpack : Localization.installJetpack)
```

Comment:
I understand from Joe's design that we want to consider this a step of the Jetpack setup flow, not login. We want to avoid giving the impression that the user has to log in twice, first with a WP.com account and then with site credentials.

Reply:
The title says `Log in to your store` and the subtitle also starts with `Log in to ...` 😆